collagraph/collagraph.py

```diff
@@ -23,6 +23,12 @@
 def create_element(type, props=None, *children) -> VNode:
     """Create an element description, based on type, props and (optionally) children"""
     key = props.pop("key") if props and "key" in props else None
+    if len(children) == 1:
+        # Check if childrenis 1 callable item: => default slot
```

Comment:
I wonder if you're supposed to also deal with the "implicit content for default slot" situation here, when receiving <template> nodes.

Reply:
Actually, in a render function based class component, you can't even express this because you have to either provide a dictionary, or a single callable. So you can't mix those. This example is purely syntactic sugar within the template. I added a task for implementing the cgx slot syntax in #19 .